protocol/scripts/governance/bridge_vesters_set_rewards.py

```diff
@@ -92,9 +94,23 @@ def estimate_block_height(current_block_height, current_block_timestamp, average
     AVG_BLOCK_TIME_FOR_ESTIMATE,
     UPDATE_2_TIME_UTC,
 )
+# Get current time in UTC
+current_utc_time = datetime.now(pytz.utc)
+# Add 4 days to the current time - use this as rough estimate for proposal pass time.
+estimated_proposal_pass_time = current_utc_time + timedelta(days=4)
```

Comment:
should we make the `timedelta` a configurable? so it can be used for testnets and other networks that have a diff gov voting period more easily?

Reply:
Done